docs/topics/preview_support.md

```diff
@@ -44,6 +45,31 @@ roborazzi {
 }
 ```
 
```

Comment:
Thanks. We might need to maintain the `build.gradle` version, and we have to check if it works. It will be a little difficult to maintain. How about something like this?

> [!NOTE]
> If you are using `build.gradle` instead of `build.gradle.kts`, you need to use the set method for each assignment, 
> like `generateComposePreviewRobolectricTests.packages.set(["com.example"])`.

Reply:
I think that is a good idea.

Should I remove my groovy code that I added in this PR, and replace it with your note  ?

Reply:
change the description like this


You can customize the generated test by adding the following configuration to your `build.gradle.kts` file:

```kotlin
roborazzi {
  @OptIn(ExperimentalRoborazziApi::class)
  generateComposePreviewRobolectricTests {
    enable = true
    // The package names to scan for Composable Previews.
    packages = listOf("com.example")
    // robolectricConfig will be passed to Robolectric's @Config annotation in the generated test class.
    // See https://robolectric.org/configuring/ for more information.
    robolectricConfig = mapOf(
      "sdk" to "[32]",
      "qualifiers" to "RobolectricDeviceQualifiers.Pixel5",
    )
    // If true, the private previews will be included in the test.
    includePrivatePreviews = true
    // The fully qualified class name of the custom test class that implements [com.github.takahirom.roborazzi.ComposePreviewTester].
    testerQualifiedClassName = "com.example.MyCustomComposePreviewTester"
  }
}
```

> [!NOTE]
> If you are using build.gradle instead of build.gradle.kts, you need to use the set method for each assignment,
like generateComposePreviewRobolectricTests.packages.set(["com.example"]).

Reply:
Looks good. It's a very small point, but it might be better to use a `code block`, though.

Reply:
@takahirom 
https://github.com/takahirom/roborazzi/pull/488/commits/c6af7bd90f97b33a08b4cd539be6fde7d3973a59

Thanks, I committed the changes using code block.

Reply:
@takahirom 
Thanks, I committed the changes using code block.

https://github.com/takahirom/roborazzi/pull/488#:~:text=add%20note%20sentence-,6db7528,-YusukeMoriJapan%20force%2Dpushed